CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - When passed a global command with the same name as a file in the current directory,
   Rustwide will now execute the global command instead of the file.
 
+### Changed
+
+- `winapi` is no longer required on unix; `nix` is no longer required on windows.
```

Comment:
The changelog is under the wrong release.

Reply:
Oops, fixed.